aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -6,8 +6,14 @@
 
 #include "ATen/Dispatch.h"
 #include "ATen/Parallel.h"
-#include "ATen/optional.h"
 #include "ATen/cpu/vec256/vec256.h"
+#include "ATen/optional.h"
+
+#ifdef __PPC64__
+using default_partitioner_type = tbb::simple_partitioner;
+#else
+using default_partitioner_type = tbb::affinity_partitioner;
+#endif
```

Comment:
Can we avoid repeating that?

Reply:
Also, looks like there's another `parallel_for` in this file. Why aren't we using the one you added in this PR?

Reply:
I've done all of this before. At some point there was an ASAN bug in ReduceOpsKernel and UnaryOpsKernel. I want to do this one step at a time and with rigorous testing.